server/commands/documentCollaborativeUpdater.ts

```diff
@@ -18,7 +19,10 @@ export default async function documentCollaborativeUpdater({
   return sequelize.transaction(async (transaction) => {
     const document = await Document.unscoped()
       .scope("withState")
-      .findByPk(documentId, {
+      .findOne({
```

Comment:
This change makes a big difference – `findByPk` overrides the scope and ignores our `"withState"` scope 😢 – this means that we've been writing the full state to disk on every persistence event. Should improve database load considerably.